python/Ganga/GPIDev/Adapters/IBackend.py

```diff
@@ -475,10 +472,6 @@ def master_updateMonitoringInformation(jobs):
                     temp_block = []
 
                 for this_block in monitorable_blocks:
-
-                    if monitoring_component and not monitoring_component.isEnabled(False) or not monitoring_component:
```

Comment:
For now with this line I'm thinking that I will add a line to the beginning of the function which checks if the monitoring was running when the function started. At the point of the line above it will become "was the monitoring running when the function started and has it now stopped? If so, `break`"